django_gruveo/settings.py

```diff
@@ -118,3 +118,5 @@
 # https://docs.djangoproject.com/en/1.11/howto/static-files/
 
 STATIC_URL = '/static/'
+
```

Comment:
@fichie23 let's not put unneeded settings here.

Reply:
I guess there is no harm to leave it, or removed it, i just leave it as default because i saw other example django apps did the same and i guess it will be more risky to removed it.

Reply:
just want to clarify this, the apps cannot working if i'm remove this line.